cmd/acr/purge.go

```diff
@@ -42,27 +41,31 @@ const (
   - Delete all tags older than 1 day in the example.azurecr.io registry inside the hello-world repository using the credentials found in 
     the C://Users/docker/config.json path
 	acr purge -r example --filter "hello-world:.*" --ago 1d --config C://Users/docker/config.json
-`
 
-	defaultNumWorkers       = 6
+  - Delete all tags older than 1 day in the example.azurecr.io registry inside the hello-world repository, with 4 purge tasks running concurrently
+	acr purge -r example --filter "hello-world:.*" --ago 1d --task-number 4
```

Comment:
>--task-number [](start = 57, length = 13)

The name `task-number` is so weird. Here are some suggestions:
- `ab` has `-c concurrency`.
- `parallel` has `-j maxjobs`
- `xargs` has `-P max-procs, --max-procs=max-procs`